### PR TITLE
ASC-1332 Refactor 'test_verify_networks_created'

### PR DESCRIPTION
**Pull request by ryan-rs**
Updated the test to utilize the OpenStack Python API for validation.

### CODE REVIEW

File: molecule/default/tests/test_verify_networks_created.py [hunk 1]
```diff
@@ -1,39 +1,44 @@
...
+    """
 
-@pytest.mark.test_id('d7fc646b-432a-11e8-b858-6a00035510c0')
-@pytest.mark.jira('asc-239')
-def test_verify_network_list(host):
-    """Verify the neutron network was created"""
-    cmd = "{} openstack network list'".format(utility_container)
-    output = host.run(cmd)
-    networks = ['GATEWAY_NET', 'PRIVATE_NET']
+    expected_networks = ('GATEWAY_NET', 'PRIVATE_NET')
```
Comment (phongdly) [inline molecule/default/tests/test_verify_networks_created.py]:
```suggestion
    vars = host.ansible('include_vars',
                        'file=./vars/main.yml')['ansible_facts']
     expected_networks = (vars['gateway_network'], vars['private_network'])
```

File: molecule/default/tests/test_verify_networks_created.py [hunk 2]
```diff
@@ -1,39 +1,44 @@
...
+@pytest.mark.jira('ASC-239', 'ASC-1332')
+def test_verify_subnet_list(os_api_conn):
+    """Verify that subnets were created.
+
+    Args:
+        os_api_conn (openstack.connection.Connection): An authorized API
+            connection to the 'default' cloud on the OpenStack infrastructure.
+    """
+
+    expected_subnets = ('GATEWAY_NET_SUBNET', 'PRIVATE_NET_SUBNET')
```
Comment (phongdly) [inline molecule/default/tests/test_verify_networks_created.py]:
```suggestion
        vars = host.ansible('include_vars',
                        'file=./vars/main.yml')['ansible_facts']
     expected_networks = (vars['gateway_network_subnet'], vars['private_network_subnet'])
     # The 'gateway_network_subnet' and 'private_network_subnet' vars are not in ./vars/main.yml file yet, need to be added
```

File: molecule/default/tests/test_verify_networks_created.py (hunk 1 above)
Comment (ryan-rs) [reply molecule/default/tests/test_verify_networks_created.py]: Updated to utilize the 'openstack_properties' fixture.

File: molecule/default/tests/test_verify_networks_created.py (hunk 2 above)
Comment (ryan-rs) [reply molecule/default/tests/test_verify_networks_created.py]: @phongdly Updated to utilize the 'openstack_properties' fixture. Also, I added the subnets to the fixture.

**Pull request merged**